[2025-08-24T17:36:04.110+0100] {processor.py:161} INFO - Started process (PID=25413) to work on /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:36:04.110+0100] {processor.py:830} INFO - Processing file /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-08-24T17:36:04.111+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:36:04.110+0100] {dagbag.py:545} INFO - Filling up the DagBag from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:36:04.141+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:36:04.141+0100] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-08-24T17:36:04.141+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:36:04.141+0100] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-08-24T17:36:04.141+0100] {processor.py:842} WARNING - No viable dags retrieved from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:36:04.167+0100] {processor.py:183} INFO - Processing /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.058 seconds
[2025-08-24T17:37:26.678+0100] {processor.py:161} INFO - Started process (PID=25477) to work on /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:37:26.679+0100] {processor.py:830} INFO - Processing file /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-08-24T17:37:26.679+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:37:26.679+0100] {dagbag.py:545} INFO - Filling up the DagBag from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:37:26.711+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:37:26.711+0100] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-08-24T17:37:26.711+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:37:26.711+0100] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-08-24T17:37:26.711+0100] {processor.py:842} WARNING - No viable dags retrieved from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:37:26.737+0100] {processor.py:183} INFO - Processing /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2025-08-24T17:39:19.536+0100] {processor.py:161} INFO - Started process (PID=25574) to work on /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:39:19.537+0100] {processor.py:830} INFO - Processing file /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-08-24T17:39:19.538+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:39:19.538+0100] {dagbag.py:545} INFO - Filling up the DagBag from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:39:19.569+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:39:19.569+0100] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-08-24T17:39:19.569+0100] {logging_mixin.py:188} INFO - [2025-08-24T17:39:19.569+0100] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-08-24T17:39:19.569+0100] {processor.py:842} WARNING - No viable dags retrieved from /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-08-24T17:39:19.595+0100] {processor.py:183} INFO - Processing /Users/homeaccount/miniconda3/envs/churn_pipeline/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
